src/routes/my/myReviewGET.js

```diff
@@ -48,18 +49,13 @@ module.exports = async (req, res) => {
         }
       });
 
-      let responseData = [];
-      responseData = myReviewArr;
+      let responseData = myReviewArr;
       responseData = duplicatedDataClean(responseData, 'reviewId', 'image');
 
       responseData = await Promise.all(
         responseData.map(async (value) => {
           let shopId = value.shopId;
           let shop = await shopDB.getShopByShopId(client, shopId);
```

Comment:
이 부분에선 변수 재할당이 후에 이뤄지고 있지 않아서 const로 바꿔도 좋을 것 같아요!
```suggestion
          const shopId = value.shopId;
          const shop = await shopDB.getShopByShopId(client, shopId);
```